CMakeLists.txt

```diff
@@ -188,8 +189,10 @@ if (APPLE)
 	set(CPACK_SOURCE_GENERATOR "")
 	set(CPACK_GENERATOR "Bundle")
 	set(CMAKE_XCODE_ATTRIBUTE_MARKETING_VERSION "${PROJECT_VERSION}")
+elseif(WIN32)
+	set(CPACK_GENERATOR "NSIS64")
```

Comment:
please leave this within the else statement. nsis isn't specific for just windows.

Reply:
You got it around backwards. I made the difference because the others fail on windows. I could add NSIS to the else ALSO, though.

Reply:
TGZ, ZIP should work on windows provided you've got that installed ofc

Reply:
Fair enough.